bundles/AdminBundle/Controller/Admin/DataObject/DataObjectController.php

```diff
@@ -2176,98 +2166,6 @@ public function previewAction(Request $request)
         return $this->redirect($urlParts['path'] . '?pimcore_object_preview=' . $id . '&_dc=' . time() . (isset($urlParts['query']) ? '&' . $urlParts['query'] : ''));
     }
 
-    /**
-     * @param  DataObject\Concrete $object
-     * @param  array $toDelete
-     * @param  array $toAdd
-     * @param  string $ownerFieldName
-     */
-    protected function processRemoteOwnerRelations($object, $toDelete, $toAdd, $ownerFieldName)
-    {
-        $getter = 'get' . ucfirst($ownerFieldName);
-        $setter = 'set' . ucfirst($ownerFieldName);
-
-        foreach ($toDelete as $id) {
-            $owner = DataObject::getById($id);
-            //TODO: lock ?!
-            if (method_exists($owner, $getter)) {
-                $currentData = $owner->$getter();
-                if (is_array($currentData)) {
-                    for ($i = 0; $i < count($currentData); $i++) {
-                        if ($currentData[$i]->getId() == $object->getId()) {
-                            unset($currentData[$i]);
-                            $owner->$setter($currentData);
-                            $owner->setUserModification($this->getAdminUser()->getId());
-                            $owner->save();
```

Comment:
Another thing which came to my mind: will it still trigger DataObject events such as preUpdate and postUpdate on the owner object? Maybe we should just come up with a solution that generates getters&setters but leaves the original saving and loading logic untouched ?